pyproject.toml

```diff
@@ -19,7 +19,7 @@ python = "^3.7"
 luigi = "*"
 boto3 = "*"
 slack-sdk = "^3.0"
-pandas = "<1.2"
+pandas = "^1.0"
```

Comment:
Since `pandas` needs Python >=3.7.1, you should specify python version as `python = "^3.7.1"`

https://pypi.org/project/pandas/